### ISSUE TITLE
ActionKeyNames called with AnyDevice mask is not returning for all devices

### ISSUE THREAD

**Issue by harbdog** [quasilyte/ebitengine-input#26]
According to the doc using the AnyDevice mask from the ActionKeyNames function should return key names for all devices, but it is currently only returning keys for keyboard and mouse devices, but not gamepad. Adding the following to the [Init code of the configfile example](https://github.com/quasilyte/ebitengine-input/blob/f6425e749344e9143132f014fe2bb76b3af4c67f/_examples/configfile/main.go#L157) shows the problem:

```golang
        ...
	g.inputHandler = g.inputSystem.NewHandler(0, keymap)

	for a := range keymap {
		actionKey := actionString(a)
		fmt.Printf("%s: %v\n", actionKey, g.inputHandler.ActionKeyNames(a, input.AnyDevice))
	}
```

They keymap.json used in the example has some gamepad bindings, but they are not shown as expected of the AnyDevice mask
```
Restart: [ctrl+r]
Secret: [ctrl+shift+q]
Left: [left a]        
Right: [right d]      
Pause: [space]   
```

**Comment by harbdog** [comment]
@quasilyte I've found where the problem is, it appears to only want to show the gamepad key names when a [gamepad is connected](https://github.com/quasilyte/ebitengine-input/blob/f6425e749344e9143132f014fe2bb76b3af4c67f/handler.go#L137). 

```golang
gamepadConnected := h.GamepadConnected()
...
	case keyGamepad, keyGamepadLeftStick, keyGamepadRightStick, keyGamepadStickMotion:
		enabled = gamepadConnected && (mask&GamepadDevice != 0)
```

While this may make sense for some cases, in my case I'm wanting to use it to export all configuration to json after allowing in-game remapping, that can later be imported as done in the "configfile" example.

**Comment by quasilyte** [comment]
This gamepadConnected filter is redundant.

The `DefaultInputMask` already tries to provide the same functionality. There is no point in duplicating this logic in ActionKeyNames function too.

So I guess it should just respect the given mask and return all keys if AnyDevice is used.

**Comment by quasilyte** [comment]
Would #31 solve this issue?

**Comment by quasilyte** [comment]
Sorry for the late response. Sometimes the notifications just find their way to bypass me.

**Comment by harbdog** [comment]
@quasilyte yes the code from PR #31 appears to resolve the issue:

```bash
This PR appears to have fixes the issue #26 when the 👍 

```bash
$ go run ./_examples/configfile/main.go 
Pause: [space gamepad_start]
Restart: [ctrl+r gamepad_back]
Secret: [ctrl+shift+q]
Left: [left a gamepad_left]
Right: [right d gamepad_right]
```
```
